lib/matplotlib/tests/test_preprocess_data.py

```diff
@@ -205,42 +158,6 @@ def func_replace_all(ax, x, y, ls="x", label=None, w="NOT"):
         func_replace_all(None, x="a", y="b", w="x", label="text", data=data) ==
         "x: [1, 2], y: [8, 9], ls: x, w: xyz, label: text")
 
-    @_preprocess_data(label_namer="y")
-    def func_varags_replace_all(ax, *args, **kwargs):
```

Comment:
See above re: "implicit" label_namer.  Otherwise this test function is redundant with func_replace_all just above.